<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loadMongoDBDialogBase</class>
 <widget class="QDialog" name="loadMongoDBDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load MongoDB Points</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Connection</string>
    </attribute>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>501</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Close_button">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>450</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>611</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Database:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Server:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Geometry field:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QComboBox" name="dbName">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="serverName">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QComboBox" name="geom_field">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QComboBox" name="filter_field">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>641</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="createFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_collection">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="listCol">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>611</width>
        <height>161</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>210</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Items</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Geometry</string>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="statusTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="load_field">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QListWidget" name="view_all">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="distinct_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>380</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Distinct</string>
     </property>
    </widget>
    <widget class="QListWidget" name="view_distinct">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>171</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="set_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>114</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>115</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MongoDB Loader</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serverName</tabstop>
  <tabstop>dbName</tabstop>
  <tabstop>geom_field</tabstop>
  <tabstop>createFile</tabstop>
  <tabstop>load_collection</tabstop>
  <tabstop>listCol</tabstop>
  <tabstop>Close_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Close_button</sender>
   <signal>clicked()</signal>
   <receiver>loadMongoDBDialogBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
